testing/e2e/clients/aks.go

```diff
@@ -357,11 +357,11 @@ func (a *aks) waitStable(ctx context.Context, objs []client.Object) error {
 					getLogsFn := func() error { // right now this just dumps all logs on the pod, if we eventually have more logs
 						// than can be stored we will need to "stream" this by using the --since-time flag
 						if err := a.runCommand(ctx, armcontainerservice.RunCommandRequest{
-							Command: to.Ptr(fmt.Sprintf("kubectl logs job/%s -n %s", obj.GetName(), ns)),
+							Command: to.Ptr(fmt.Sprintf("kubectl logs job/%s -n %s --since=5m", obj.GetName(), ns)),
```

Comment:
We shouldn't do this. The file is fine. The probably is on the GitHub action runner so change the workflow to tail instead of cat the entire file.

We should continue to gather the entire Job's output into this file.

Reply:
also we don't just want to tail. we should print the first 50 lines or so. that shows the testing order and other startup context

Reply:
done